arm-asazure/2016-05-06/swagger/asazure.json

```diff
@@ -0,0 +1,553 @@
+      "required": [
+        "location",
+        "sku",
+        "tags",
+        "properties"
+      ]
+    },
+    "AnalysisServicesResourceParameters": {
+      "description": "Provision request specification",
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Location of the Analysis Services resource"
+        },
+        "sku": {
+          "$ref": "#/definitions/ResourceSkuTier",
+          "description": "Sku of the Analysis Services resource"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": { "type": "string" },
+          "description": "Key value pairs of additional properties that can ebe specified"
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AnalysisServicesResourceProperties",
+          "description": "Provision request properties"
+        }
+      }
+    },
+    "AnalysisServicesResource": {
+      "description": "Represents an instance of Analysis Services resource",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Identifier that represents the Analysis Services resource"
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of the Analysis Services resource"
+        },
+        "resourceType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Specifies the type of Analysis Services resource"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalysisServicesResourceParameters"
+        }
+      ],
+      "x-ms-azure-resource": true
+    },
+    "AnalysisServicesResources": {
+      "description": "A collection of Analysis Services resources",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/AnalysisServicesResource" },
+          "description": "The collection of Analysis Services resources"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AnalysisServicesResourceProperties": {
+      "description": "Properties of Analysis Services resource",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The current deployment state of Analysis Services resource"
+        },
+        "serverFullName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Full name of the Analysis Services resource"
+        },
+        "asAdministrators": {
+          "$ref": "#/definitions/ServerAdministrators",
+          "description": "List of users that are administrators on the Analysis Services resource"
+        }
+      }
+    },
+    "ResourceSkuTier": {
```

Comment:
This should be named as "ResourceSku". It has two properties named "name" and "tier". It gets confusing. Initially I thought, the object only has one property named tier hence you named it like that.